.github/actions/publish-npm/action.yml

```diff
@@ -19,11 +19,15 @@ runs:
       with:
         aws_assume_role: ${{ inputs.aws_assume_role }}
         ssm_parameter_pairs: '/production/common/releasing/npm/token = NODE_AUTH_TOKEN'
-    - name: publish
+    - uses: ./.github/actions/ci
+      name: 'Run CI'
+      env:
+        NODE_AUTH_TOKEN: ${{ env.NODE_AUTH_TOKEN }}
```

Comment:
Does publint use the token? Usually I try to avoid using the tokens in non-publish runs.

Reply:
The problem I was trying to solve is that actions/setup-node needs the token to setup auth with the registry. But I'll try to refactor this.

Reply:
I've updated so that the CI step runs _before_ the release-secrets step, so it won't receive the token.